[SDK-163] Add customization guide (#29)
readyplayerme/rpm-unity-sdk-avatar-creator: Documentation~/CustomizationGuide.md

# Customization Guide

> This guide will help you to customize the Ready Player Me avatar creator to fit your needs. The UI is build in mind with separation between logic and UI.

Reviewer: wording improvement suggestion
```suggestion
This guide will help you to customize the Ready Player Me avatar creator to fit your needs. The UI was designed in a way that separates the logic from the UI elements.
```

## Requirements

> You are allowed to change the entire UI, the only thing that you are required to have in your custom implementation is the Ready Player Me sign-in button and Ready Player Me account-creation UI. This is a legal requirement from Ready Player Me.

Reviewer: I think we should add some screenshots of these UI elements, like Gevorg did for Unreal guide as they are quite important and it will make it clear which ones are required.

## Creating a Custom UI From Scratch

All required APIs are provided in the package. They can be used to create a new custom UI. The package also contains a sample UI that can be used as a reference. The following are the most important APIs that you will required.
- AuthManager - Requests for handling authentication.
- AvatarManager - Requests for creating, updating and deleting avatars.
- PartnerAssetManager - Requests for loading partner assets.

## Customizing the Sample

>  The sample comes with a default UI similar to the Web avatar creator. The UI is built using uGUI. All major UI components such as screen, asset buttons, asset type panels are prefabs, and can be edited easily. For detailed description of structure of the sample please see the SampleStructure.md.

Reviewer: You can link the sample structure file here. Not sure if my code suggestion will work though :D 
```suggestion
 The sample comes with a default UI similar to the Web avatar creator. The UI is built using uGUI. All major UI components such as screen, asset buttons, asset type panels are prefabs, and can be edited easily. For detailed description of structure of the sample please see the [SampleStructure](CustomizationGuide.md).
```

### Changing background color
A small demonstration on how to change the background color of different screens. 

> https://github.com/readyplayerme/rpm-unity-sdk-avatar-creator/assets/1121080/ae412932-1bd9-4d00-b6b5-6525adecf9c7

Reviewer: The videos you included are great however I reckon we should still provide some written steps if possible. 
Having both would make it super easy to follow.
